freeipa.spec.in

```diff
@@ -590,6 +590,15 @@ Requires: cifs-utils
 This package provides command-line tools to deploy Samba domain member
 on the machine enrolled into a FreeIPA environment
 
+%package client-epn
+Summary: Tools to configure Expired Password Notification in IPA
+Group: System Environment/Base
+Requires: %{name}-client = %{version}-%{release}
+
+%description client-epn
+This package provides a service to collect and send via SMTP expired
```

Comment:
EPN stands for Expiring Password Notification.

Reply:
Fixed, thanks